test/e2e/framework/test_context.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"k8s.io/klog"
+
+	"sigs.k8s.io/kubefed/pkg/controller/util"
+)
+
+type TestContextType struct {
+	KubeConfig                      string
+	KubeContext                     string
+	KubefedSystemNamespace          string
+	ReportDir      					string
+	ReportPrefix   					string
+	SingleCallTimeout               time.Duration
+	LimitedScope                    bool
```

Comment:
This line and the 2 that follow are likely not necessary.

Reply:
Good catch, lots of cruft.